waspc/src/Wasp/AppSpec/Valid.hs

```diff
@@ -165,7 +166,7 @@ validateOnlyEmailOrUsernameAndPasswordAuthIsUsed spec =
 validateDbIsPostgresIfPgBossUsed :: AppSpec -> [ValidationError]
 validateDbIsPostgresIfPgBossUsed spec =
   [ GenericValidationError
-      "The database provider in the schema.prisma file must be \"postgresql\" since there are jobs with executor set to PgBoss."
+      ("The database provider in the schema.prisma file must be \"" ++ Psl.Db.dbToPrismaProviderId AS.Db.PostgreSQL ++ "\" since there are jobs with executor set to PgBoss.")
```

Comment:
It still makes no sense here to take something that is AppSpec concept (AS.Db.PostgreSQL) and then map it to Prisma concept.

We are in this error message specifically telling them that in schema.prisma file, provider must be "postgresql". This has nothing to do with Wasp's AppSpec and how we describe the database there.

What instead you should do is define something like `pslPostgresqlKeyword :: String` or `pslPostgresqlIdentifier :: String` or whatever is the correct term in Prisma's world, and have it have value of `"postgresql"`, and use that here, that is it. You would define that somewhere in Wasp.Prisma.PSL most likely (I didn't check the exact name of modules but I am guessing it goes something like this).